metadata.go

```diff
@@ -64,7 +64,6 @@ type FunctionMetadata struct {
 	Name              string
 	ArgumentTypes     []TypeInfo
 	ArgumentNames     []string
-	Body              string
```

Comment:
Trying to understand: why there is a difference in these table's schema in yugabytedb versus Cassandra?
I could not find a version of cassandra which did not have these columns on the system_schema tables mentioned here.

Reply:
In CQL, in the future we can investigate how to implement the `body` and `return_type` as virtual columns.

For gocql, let's do this simplification.